uiowa/uiowa
# Find Text - basic permissions test

Adds additional functionality to the Find Text module (taxonomy term searching) as well as several tests to aid in future development.

Resolves #4589 





# How to test



- `git checkout find-text-tests && git pull`
- Manually run the `sitenow_find_text` tests that have been added
   - https://github.com/uiowa/uiowa/tree/main/tests
   - All tests should pass. For extra thoroughness, you can "break" functionality and check that the tests no longer work. For instance, rename the page permission in the permissions yml, break a case statement in `get_text_field_tables` and see that the related entity tests break, etc.
- Check that the latest Travis CI build passed successfully
- Sync a site that has taxonomy terms, or create some new taxonomy terms
- Try searching for terms that will match taxonomy term names and/or descriptions, see that they appear in the results
- Try searching for a menu and node content, and see that they still appear in the results
